fdbserver/TagPartitionedLogSystem.actor.cpp

```diff
@@ -2248,19 +2244,18 @@ ACTOR Future<Void> TagPartitionedLogSystem::epochEnd(Reference<AsyncVar<Referenc
 						Reference<LogSet> logSet = lockResults[0].logSet;
 						if (versionRepCount[k] >=
 						    std::get<1>(result.unknownCommittedVersions[i]) - logSet->tLogReplicationFactor + 1) {
-							if (k > rvLogs[result.id] && k > knownCommittedVersion) {
+							Version rv = rvLogs.find(result.id) != rvLogs.end() ? rvLogs[result.id] : 0;
```

Comment:
Sorry, why is it not "k >= knownCommittedVersion" below?

Reply:
This logic was from the [previous PR](https://github.com/apple/foundationdb/pull/5949). We skip entries in the unknownCommittedVersions list less than or equal to the max(KCV) that have not been pruned, as we know those versions have been committed. It is inclusive of max(KCV) because the KCV has been committed (by definition)

Reply:
Ok, makes sense. Thanks!